Purchases/Public/RCPurchases.m

```diff
@@ -460,7 +404,8 @@ - (void)identify:(NSString *)appUserID completionBlock:(nullable RCReceivePurcha
 }
 
 - (void)resetWithCompletionBlock:(nullable RCReceivePurchaserInfoBlock)completion {
-    [self.userDefaults removeObjectForKey:[self attributionDataUserDefaultCacheKeyForAppUserID:self.appUserID]];
+    [self.deviceCache setLatestNetworkAndAdvertisingIdsSent:nil
```

Comment:
This will call `self.userDefaults setObject:nil` instead of calling `removeObjectForKey`. Is it the same to set a nil object than calling `removeObjectForKey`?

Reply:
it does in practice, but I can't find any specific docs pointing to it. I'll update to `removeObjectForKey` to be safe.